.github/CODEOWNERS

```
@@ -1,5 +1,5 @@
-# Team responsable for Fleet Server
-* @elastic/elastic-agent-control-plane
+# Team responsible for Fleet Server
+* @elastic/fleet
```

```suggestion
* @elastic/fleet @elastic/elastic-agent-control-plan
```

It may make sense to keep both teams in here, or create a new team group of owners of developers who frequently work on fleet server.

Not opposed to switching it over to `elastic/fleet` but if you have auto review assignment turned on it may start pinging developers that have never worked on fleet server for reviews.

---

I am more concerned about making sure the right people are pinged for reviews automatically than exactly what the codeowners file says.

---

Usually I'm not keen to have individuals on these files. It makes maintaining it harder, usually people move more than teams change responsibilities.

---

then it might make more sense to add back control-plane team on main, and backport to 8.5 and 8.6

---

raised https://github.com/elastic/fleet-server/pull/2117